migrations/20211011151008_user.js

```diff
@@ -1,16 +1,16 @@
-exports.up = function(knex) {
-    return knex
-        .raw(`
+exports.up = function (knex) {
+  return knex
+    .raw(`
             GRANT INSERT ON duplicates TO resolver;
+            GRANT ALL ON SEQUENCE duplicates_id_seq to resolver;
```

Comment:
Did work out what this is?

Reply:
I think it allows a user role to GET the sequence of duplicate entries when it needs to insert a new entry to ensure they are in the right order. I'm not 100% sure. I didn't test this without but just in case copied the format elsewhere just in case it created any issues.